week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -15,12 +15,30 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 
 function sortAges(arr) {
+ let maxarray = [];
+ for ( let i = 0 ; i <= arr.length ; ++i ){
+   if (typeof arr[i] == `number`){
+     maxarray.push(arr[i]);
+   }
+ }
+ sortAge(maxarray);

```

Comment:
The name of this function is very similar to the one above and could be easily confused. You can try renaming to avoid confusion. In the previous part you already removed any non-numeric values.